go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -61,15 +64,17 @@ var (
 	}
 
 	registerCmd = &cobra.Command{
-		Use:   cmdRegister,
-		Short: "register an entity",
-		Run:   doRegisterOrDeregister,
+		Use:     cmdRegister,
+		Aliases: []string{"register"},
+		Short:   "generate a register entity transaction",
+		Run:     doRegisterOrDeregister,
 	}
 
 	deregisterCmd = &cobra.Command{
-		Use:   "deregister",
-		Short: "deregister an entity",
-		Run:   doRegisterOrDeregister,
+		Use:     "gen_deregister",
```

Comment:
I know this isn't your fault, but it seems inconsistent, `cmdRegister` is a constant but this one is not? Should be either IMO.

Reply:
I'll make them both constants.

Reply:
Actually I decided to use literals. Thoughts on the last commit @kostko?